schematic/store/synapse.py

```diff
@@ -2128,27 +2145,24 @@ def _add_id_columns_to_manifest(
 
             # Rename `Uuid` column if it wasn't specified in the schema
             if col_in_dataframe("Uuid", manifest) and not uuid_col_in_schema:
-                manifest.rename(columns={"Uuid": "Id"}, inplace=True)
+                manifest = manifest.rename(columns={"Uuid": "Id"})
             # If no `Uuid` column exists or it is specified in the schema, create a new `Id` column
             else:
                 manifest["Id"] = ""
 
-        # Retrieve the ID column name (id, Id and ID) are treated the same.
-        id_col_name = [col for col in manifest.columns if col.lower() == "id"][0]
-
-        # Check if values have been added to the Id coulumn, if not add a UUID so value in the row is not blank.
+        # Fill in UUIDs in the "Id" column if missing
         for idx, row in manifest.iterrows():
-            if not row[id_col_name]:
+            if not row["Id"]:
                 gen_uuid = str(uuid.uuid4())
-                row[id_col_name] = gen_uuid
-                manifest.loc[idx, id_col_name] = gen_uuid
+                row["Id"] = gen_uuid
+                manifest.loc[idx, "Id"] = gen_uuid
 
-        # add entityId as a column if not already there or
-        # fill any blanks with an empty string.
-        if not col_in_dataframe("entityId", manifest):
```

Comment:
similarly it here, I changed this to `if "entityId" not in manifest:` because at this point, the column should have been normalized.